.github/ISSUE_TEMPLATE/accessibility.md

```diff
@@ -2,10 +2,11 @@
 name: Accessibility
 about: Report an accessibility issue
 title: ''
-labels: 'accessibility-awaiting-triage'
+labels: ''
```

Comment:
Would it be possible to put the changes in the labels into a different PR, if they were intentional?

I'm happy to fight with git with you, if you'd like!